crates/ruff/src/rules/pycodestyle/snapshots/ruff__rules__pycodestyle__tests__E501_E501.py.snap

```diff
@@ -10,11 +10,11 @@ E501.py:5:89: E501 Line too long (123 > 88 characters)
 8 | """
   |
 
-E501.py:16:89: E501 Line too long (95 > 88 characters)
+E501.py:16:85: E501 Line too long (95 > 88 characters)
```

Comment:
We could use this as a chance to improve the diagnostic. Any recommendations for a replacement term for `characters` that captures that we compare the width and not characters count?